src/index.ts

```diff
@@ -1,29 +1,36 @@
+import { Orderbook, SupportedProvider } from '@0x/asset-swapper';
 import { WSClient } from '@0x/mesh-rpc-client';
+import { RPCSubprovider, Web3ProviderEngine } from '@0x/subproviders';
+import { providerUtils } from '@0x/utils';
 import * as express from 'express';
 
 import * as config from './config';
 import { SRA_PATH } from './constants';
 import { getDBConnectionAsync } from './db_connection';
+import { DummyOrderProvider } from './dummy_order_provider';
 import { logger, logMiddleware } from './logger';
 import { OrderWatcherService } from './services/order_watcher_service';
 import { OrderBookService } from './services/orderbook_service';
 import { SRAHttpService } from './services/sra_http_service';
 import { StakingDataService } from './services/staking_data_service';
 import { StakingHttpService } from './services/staking_http_service';
+import { SwapQuoteHttpService } from './services/swap_quote_service';
 import { WebsocketService } from './services/websocket_service';
+import { OrderStoreDbAdapter } from './utils/order_store_db_adapter';
+
+const createWeb3Provider = (rpcHost: string): SupportedProvider => {
```

Comment:
Are you worried about requests failing due to sporadic downstream failures from alchemy?

Reply:
We can use `RedundantSubprovider` to add some retry logic

Reply:
I actually had some weird `setEngine` issues when using `RedundantSubprovider`, we could add it in as a TODO. 

This something we should think carefully about. If Alchemy timeouts (say `30s`) then it tries INFURA (`1s`) then this quote is now `31s` old before it gets returned to the user.